oneview_redfish_toolkit/tests/blueprints/test_computer_system.py

```diff
@@ -581,3 +607,296 @@ def test_remove_computer_system_task_error(
+        g.oneview_client.server_hardware.get.assert_has_calls(
+            [
+                call(self.sh_id),
+                call(self.spt_id),
+                call(self.drive1_id),
+                call(self.drive2_id)
+            ]
+        )
+        g.oneview_client.server_profile_templates.get.assert_has_calls(
+            [
+                call(self.sh_id),
+                call(self.spt_id),
+                call(self.drive1_id),
+                call(self.drive2_id)
+            ]
+        )
+        g.oneview_client.index_resources.get.assert_not_called()
+        g.oneview_client.server_profiles.create.assert_not_called()
+
+    @mock.patch.object(computer_system, 'g')
+    def test_create_system_when_request_content_did_not_have_storage(self, g):
+        """Tests create a redfish System without Storage. Should works well"""
+
+        g.oneview_client.server_hardware.get.side_effect = [
+            self.server_hardware,
+            self.not_found_error,
+            self.not_found_error,
+            self.not_found_error
+        ]
+        g.oneview_client.server_profile_templates.get.side_effect = [
+            self.not_found_error,
+            self.server_profile_template,
+            self.not_found_error,
+            self.not_found_error,
+            self.server_profile_template
+        ]
+        g.oneview_client.index_resources.get.side_effect = [
+            self.not_found_error,
+            self.not_found_error,
+            self.not_found_error,
+            self.not_found_error
+        ]
+        g.oneview_client.server_profiles.create.return_value = \
+            self.server_profile
+
+        response = self.client.post(
+            "/redfish/v1/Systems/",
+            data=json.dumps(self.data_to_create_system),
+            content_type='application/json')
+
+        self.assertEqual(status.HTTP_201_CREATED, response.status_code)
+        self.assertEqual("application/json", response.mimetype)
+        self.assertIn(
+            "/redfish/v1/Systems/" + self.server_profile["uuid"],
+            response.headers["Location"]
+        )
+        g.oneview_client.server_hardware.get.assert_has_calls(
+            [
+                call(self.sh_id),
+                call(self.spt_id),
+                call(self.drive1_id),
+                call(self.drive2_id)
+            ]
+        )
+        g.oneview_client.server_profile_templates.get.assert_has_calls(
+            [
+                call(self.sh_id),
+                call(self.spt_id),
+                call(self.drive1_id),
+                call(self.drive2_id),
+                call(self.spt_id)
+            ]
+        )
+        g.oneview_client.index_resources.get.assert_has_calls(
+            [
+                call('/rest/drives/' + self.sh_id),
+                call('/rest/drives/' + self.spt_id),
+                call('/rest/drives/' + self.drive1_id),
+                call('/rest/drives/' + self.drive2_id)
+            ]
+        )
```

Comment:
It seems we should add a test when invalid Computer System Resource Block. Also, there is no test when server profile creation returns an error.